crc/api/study.py

```diff
@@ -7,7 +7,7 @@
 from crc.api.common import ApiError, ApiErrorSchema
 from crc.models.protocol_builder import ProtocolBuilderStatus
 from crc.models.study import Study, StudyEvent, StudyEventType, StudyModel, StudySchema, StudyForUpdateSchema, \
-    StudyStatus
+    StudyStatus, StudyAssociatedSchema
```

Comment:
Great that we have a Schema for StudyAssociated but did you use it?

Reply:
It was breaking the test_email_cc and a few other seemingly random tests so I was planning to come back to it to figure out once the main feature was working

Reply:
I can draft this PR and figure that out now tho